distro/pom.xml

```diff
@@ -36,7 +36,7 @@
     <!-- this is request for the OCL UI -->
     <owa.version>1.14.0</owa.version>
     <attachments.version>3.0.0</attachments.version>
-    <referencedemodata.version>2.4.0-SNAPSHOT</referencedemodata.version>
+    <referencedemodata.version>2.4.0</referencedemodata.version>
```

Comment:
Hi @vasharma05, lets use 2.3.1 patch release which I've just triggered.

Reply:
2.3.1, right? Or is it something with patch  in the version as well?

Reply:
There is a bug fix coming with version 2.3.1 of the Patient Flags module.